# HG changeset patch
# User Rik <rik@octave.org>
# Date 1347560482 25200
#      Thu Sep 13 11:21:22 2012 -0700
# Node ID e4c6947d87cd89722fe25d82d843303e2b66f355
# Parent  9453acf885f26d0ec9da19cf9747ace702f07d0c
build: Use TEMPLATE_SRC variable, not OTHER_INC, for C++ template instantiation code.

* liboctave/array/module.mk, liboctave/operators/module.mk:
Use TEMPLATE_SRC variable, not OTHER_INC, for C++ template instantiation code.

diff --git a/liboctave/array/module.mk b/liboctave/array/module.mk
--- a/liboctave/array/module.mk
+++ b/liboctave/array/module.mk
@@ -125,19 +125,17 @@ ARRAY_SRC = \
   array/uint16NDArray.cc \
   array/uint32NDArray.cc \
   array/uint64NDArray.cc \
   array/uint8NDArray.cc 
 
 TEMPLATE_SRC += \
   array/Array.cc \
   array/DiagArray2.cc \
+  array/intNDArray.cc \
   array/MArray.cc \
   array/MDiagArray2.cc
 
-OTHER_INC += \
-  array/intNDArray.cc
-
 noinst_LTLIBRARIES += array/libarray.la
 
 array_libarray_la_SOURCES = $(ARRAY_SRC)
 array_libarray_la_CPPFLAGS = $(liboctave_la_CPPFLAGS)
 
diff --git a/liboctave/operators/module.mk b/liboctave/operators/module.mk
--- a/liboctave/operators/module.mk
+++ b/liboctave/operators/module.mk
@@ -34,17 +34,17 @@ OPERATORS_INC = \
   operators/mx-op-defs.h \
   operators/Sparse-diag-op-defs.h \
   operators/Sparse-op-defs.h \
   operators/Sparse-perm-op-defs.h
 
 ## There are no distributed source files in this directory
 OPERATORS_SRC =
 
-OTHER_INC += \
+TEMPLATE_SRC += \
   operators/mx-inlines.cc
 
 OP_SRCDIR = $(abs_top_srcdir)/liboctave/operators
 
 ## Special rules for sources which must be built before rest of compilation.
 $(VX_OP_INC) $(VX_OP_SRC) : operators/mk-ops.awk operators/vx-ops
 	(cd operators; $(AWK) -f $(OP_SRCDIR)/mk-ops.awk prefix=vx $(OP_SRCDIR)/vx-ops)
 
